samples/benchmark/bunnymark/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,150 @@
+            "macos" -> listOf(targetFromPreset(presets["godotMacosX64"], "macos"))
+            "android" -> if (project.hasProperty("armArch")) {
+                when(armArch) {
+                    "X64" -> listOf(targetFromPreset(presets["godotAndroidNativeX64"], "androidX64"))
+                    "arm64" -> listOf(targetFromPreset(presets["godotAndroidNativeArm64"], "androidArm64"))
+                    else -> listOf(targetFromPreset(presets["godotAndroidNativeArm64"], "androidArm64"))
+                }
+            } else listOf(targetFromPreset(presets["godotAndroidNativeArm64"], "androidArm64"))
+            "ios" -> if (project.hasProperty("armArch")) {
+                when (armArch) {
+                    "arm64" -> listOf(targetFromPreset(presets["godotIosArm64"], "iosArm64"))
+                    "X64" -> listOf(targetFromPreset(presets["godotIosX64"], "iosX64"))
+                    else -> listOf(targetFromPreset(presets["godotIosArm64"], "iosArm64"))
+                }
+            } else listOf(targetFromPreset(presets["godotIosArm64"], "iosArm64"))
+            else -> listOf(targetFromPreset(presets["godotLinuxX64"], "linux"))
+        }
+    } else {
+        listOf(
+                targetFromPreset(presets["godotLinuxX64"], "linux"),
+                targetFromPreset(presets["godotMacosX64"], "macos"),
+                targetFromPreset(presets["godotMingwX64"], "windows"),
+                targetFromPreset(presets["godotAndroidNativeArm64"], "androidArm64"),
+                targetFromPreset(presets["godotAndroidNativeX64"], "androidX64"),
+                targetFromPreset(presets["godotIosArm64"], "iosArm64"),
+                targetFromPreset(presets["godotIosX64"], "iosX64")
+        )
+    }
+}
+
+fun configureTargetAction(kotlinTarget: @ParameterName(name = "target") org.jetbrains.kotlin.gradle.plugin.KotlinTarget) {
+    kotlinTarget.compilations.getByName("main") {
+        if (this is org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeCompilation) {
+            println("Configuring target ${this.target.name}")
+            this.target.compilations.all {
+                dependencies {
+                    implementation("org.godotengine.kotlin:godot-library-extension:0.1.0-3.2")
+                    implementation("org.godotengine.kotlin:annotations:0.1.0-3.2")
+                }
+            }
+            if (project.hasProperty("iosSigningIdentity") && this.target.name == "iosArm64") {
+                tasks.build {
+                    doLast {
+                        exec {
+                            commandLine = listOf("codesign", "-f", "-s", iosSigningIdentity, "build/bin/iosArm64/releaseShared/libkotlin.dylib")
+                        }
+                        exec {
+                            commandLine = listOf("install_name_tool", "-id", "@executable_path/dylibs/ios/libkotlin.dylib", "build/bin/iosArm64/releaseShared/libkotlin.dylib")
```

Comment:
Same as the other PR. I did not see it in first place. I think I did not see it in buildType PR.
@chippmann do you think we can go with hard coded debug or release for ios signing here and make another PR to add this to all projects samples ?

Reply:
Yes i think so. As we overlooked it previously, we can merge it for now and fix it in all samples in a separate PR.